salt/states/cloud.py

```diff
@@ -444,8 +445,8 @@ def volume_detached(name, server_name=None, provider=None, **kwargs):
     if name in volumes and not volumes[name]['attachments']:
         volume = volumes[name]
         ret['comment'] = (
-            'Volume {name} is not currently attached to anything.'
-        ).format(**volumes[name])
+            'Volume {0} is not currently attached to anything.'
+        ).format(name, **volumes[name])
```

Comment:
Thanks @rallytime 
But '{name}' is throwing a KeyError: 'name'. So, it is requored to change to '{0}'.